tools/mo/unit_tests/mo/utils/mo_fallback_test.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Not sure that these tests will work this way on CI. Probably some wrapper (like used in frontend_ngraph_test.py) would be needed

Reply:
good catch, added